frontend/src/pages/RidersTable.tsx

```diff
@@ -265,9 +265,12 @@ const Table = () => {
 
   useEffect(() => {
     async function getExistingRiders() {
-      const response = await fetch('/riders');
-      const data = (await response.json());
-      let allRiders: Rider[] = data.map(function (rider: any) {
+      const ridersData = await fetch('/riders').then(res => {
+        return res.json();
+      }).then(data => {
+        return data["data"];
+      });
```

Comment:
rewrite to be single line arrow functions, and either dot notation or destructure the parameter for data